File: application/config/gallery.php
<?php defined('SYSPATH') or die('No direct script access.'); 

/**
 * Configuration for the gallery
 */
return array(
	// Directory the gallery photos are stored in
	'image_dir' => '/var/www/gallerypics/',
	// URL corresponding to the above directory
	'image_url' => 'http://localhost/gallerypics/',
	// Subdirectory for resized images
	'resized_subdir' => 'resized/',
	
	// Widths
	'thumbnail_width' => 200,
	'normal_width' => 700,
);
?>

File: Daniel15.Web/Configuration/Gallery.cs
namespace Daniel15.Web.Configuration
{
	public class Gallery
	{
		/// <summary>
		/// Gets the name of the image gallery
		/// </summary>
		public string Name { get; set; }

		/// <summary>
		/// Gets the friendly title of the image gallery
		/// </summary>
		public string Title { get; set; }

		/// <summary>
		/// Gets the URL images in this gallery are directly accessible at
		/// </summary>
		public string ImageUrl { get; set; }

		/// <summary>
		/// Gets the directory images for this gallery are stored in
		/// </summary>
		public string ImageDir { get; set; }
	}
}
